php-mode.el

```diff
@@ -1033,10 +1043,30 @@ PHP heredoc."
   "PHP Mode face used to highlight function names."
   :group 'php-faces)
 
+(defface php-function-caller-face '((t (:inherit default)))
+  "PHP Mode face used to highlight calling function names."
+  :group 'php-faces)
```

Comment:
This face highlights function/method name part of `$obj->method`.

I searched for other implementations, and I found js2-mode have `js2-function-call`. 
I will follow this.

<img width="841" alt="2017-04-07 6 25 36" src="https://cloud.githubusercontent.com/assets/822086/24794195/a9111af8-1bbf-11e7-9d5d-32303bda1fd4.png">